src/sidebar/services/groups.js

```diff
@@ -196,8 +191,25 @@ function groups(
       });
   }
 
+  const sortGroups = memoize(groups => {
+    // Sort in the following order: scoped, public, private.
+    // This is for maintaining the order of the old groups menu so when
+    // the old groups menu is removed this can be removed.
+    const worldGroups = groups.filter(g => g.id === '__world__');
+    const nonWorldScopedGroups = groups.filter(
+      g => g.id !== '__world__' && ['open', 'restricted'].includes(g.type)
```

Comment:
I don't think we need to worry since we're expecting this code to go away very soon, but I wonder if it is possible that private groups could be scoped in future.